ember-addon/index.js

```diff
@@ -0,0 +1,17 @@
+var Funnel = require('broccoli-funnel');
```

Comment:
I'm thinking if this line is move inside the `treeForVendor` function we can reference the `broccoli-funnel` from Ember-CLI on the fly?